### ISSUE TITLE
Comparison of missing driving simulators in the nocturne paper[Question] 

### ISSUE THREAD

**Issue by AizazSharif** [facebookresearch/nocturne#60]
### Question

Hi,

I was reading the nocturne [paper](https://arxiv.org/pdf/2206.09889.pdf) and came across Table 1 which describes a comparison between different autonomous driving open-source simulators. I have two queries below:

![Screenshot from 2022-10-18 10-42-46](https://user-images.githubusercontent.com/9911205/196382009-e12dc791-5922-43dd-95e7-7e2bb2f6af4a.png)

1. I wanted to mention a few of the simulators I know (due to my ongoing work in Ph.D.) which are missing in the comparison such as Flow,  LGSVL, and AirSim. They are quite famous so If there is any specific reason for not comparing them please mention it here since I am also performing comparisons of such simulators in my research.

2. For Highway-env, I think they also support multi-agent driving (at least in a cooperative manner) just like the Flow simulator, but in Table 1 it is not shown as multi-agent.
 
Any response would be highly appreciated.

**Comment by AizazSharif** [comment]
Replying here in case it got under the radar, else I can close the issue later since it is nothing technical.

**Comment by eugenevinitsky** [comment]
Hi Aiziz, thanks for the reminder! There's a few things going on here:
1) We made some judicious choices in this list due to space and just tried to list the things that felt closest to focusing on the actual development of driving agents. For this reason, we left out libraries that were more focused around traffic simulation like FLOW, VISSIM. I was one of the main developers of Flow and felt that its intent didn't really fit here. As for AirSim, we could include AirSim, but it also does seem quite focused around drones.
2) I was not aware that highway-sim now supported multi-agent as it did not the last time I looked. I will update this.

Thanks for the question and let me know if you have any additional questions!
